apps/web/src/components/hero/index.tsx

```diff
@@ -12,11 +12,22 @@ function Hero(): React.JSX.Element {
 
   const onSubmit = (e: React.FormEvent): void => {
     e.preventDefault()
-
+    
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/
```

Comment:
I think we can further improve the regex by only accepting \w\s and not everything

Reply:
What about ^[\w]+@[\w]+\.[\w]{2,}$ or ^[^\s@]+@[^\s@]+\.[\w]{2,}$ ? @rajdip-b